packages/playwright/src/takeSnapshot.test.ts

```diff
@@ -0,0 +1,116 @@
+import { dedent } from 'ts-dedent';
+import express, { type Request } from 'express';
+import { Server } from 'http';
+import { Browser, chromium, devices, Page } from 'playwright';
+
+import { chromaticSnapshots, takeSnapshot } from './takeSnapshot';
+import { TestInfo } from 'playwright/test';
+
+const TEST_PORT = 13339;
+
+const baseUrl = `http://localhost:${TEST_PORT}`;
```

Comment:
If this is trying to be a unit test of `takeSnapshot()`, then starting a server and doing all of the other setup in order to run Playwright for this feels like overkill. That could be a sign that we have too much logic tied up with the Playwright hooks. But could we test it by passing in a mocked `Page` object instead?

Reply:
Good point. I'll mock `page` and that should trim the tests down quite a bit.